analyzers/src/SonarAnalyzer.CSharp/Rules/InfiniteRecursion.cs

```diff
@@ -65,6 +65,14 @@ protected override void Initialize(SonarAnalysisContext context)
                     }
                 },
                 SyntaxKind.PropertyDeclaration);
+
+            context.RegisterSyntaxNodeActionInNonGenerated(
+                c =>
+                {
+                    var @operator = (OperatorDeclarationSyntax)c.Node;
+                    CheckForNoExitMethod(c, (CSharpSyntaxNode)@operator.Body ?? @operator.ExpressionBody, @operator.OperatorToken);
+                },
+                SyntaxKind.OperatorDeclaration);
```

Comment:
The rule is also missing infinite recursions in Constructors, Destructors, Indexers and there might be other declarations. It is on my plate to add test cases for these and report an issue for it. I am raising it here only for awareness so this PR`s scope stays limited to OperatorDeclarations.

Reply:
you can look at [UcfgGenerator](https://github.com/SonarSource/sonar-security/blob/master/frontend/csharp/src/SonarAnalyzer.Security/Rules/UcfgGenerator.cs) for all the syntax kinds for which we generate UCFGs